src/test/emulators/auth/misc.spec.ts

```diff
@@ -527,4 +581,18 @@ describeAuthEmulator("emulator utility APIs", ({ authApi }) => {
           .equals("Users are present, unable to set passthrough mode");
       });
   });
+
+  it("should error on updating usageMode on PATCH /emulator/v1/projects/{PROJECT_ID}/tenants/{TENANT_ID}/config", async () => {
```

Comment:
This whole API makes no sense. As you've observed, `usageMode` cannot be set per tenant, and I don't think `allowDuplicateEmails` can be set per tenant in production either. Shouldn't tenant inherit these two settings from agent project config?

Reply:
Yeah, that all makes sense, thanks for the clarification - I'll go ahead and revert the APIs and if they're needed later on, we can address it then 👍👍